### ISSUE TITLE
Jump to

### ISSUE THREAD

**Issue by Vages** [ZeeZide/Shrugs#19]
In the official Slack app, I can press Command+K and get this window: 
<img width="810" alt="image" src="https://user-images.githubusercontent.com/3520744/95722408-69b99e00-0c63-11eb-8f27-83699f134bb0.png">

It allows me to quickly jump to channels by typing parts of their name. Having this in Shrugs would be awesome.

**Comment by helje5** [comment]
Yes, quick-open, one of the features that lay around half-done, need to finish it up. (it is pretty similar to the completion panel, just with different UI)
